src/main/java/com/nvidia/spark/rapids/jni/CastStrings.java

```diff
@@ -109,6 +109,24 @@ public static ColumnVector toIntegersWithBase(ColumnView cv, int base,
       type.getTypeId().getNativeId()));
   }
 
+  /**
+   * Converts an integer column to a string column by converting the underlying integers to the
+   * specified base.
+   *
+   * Note: Right now we only support base 10 and 16. The hexadecimal values will be
+   * returned without leading zeros or padding at the end
+   * 
+   * Example:
+   * input = [123, -1, 0, 27, 342718233]
+   * s = fromIntegersWithBase(input, 16)
+   * s is [ '4D2', 'FFFFFFFF', '0', '1B', '146D7719']
```

Comment:
is this accurate? 123 should map to "7B"

Reply:
Yes, you are right. I copied this from cudf doc [here](https://github.com/rapidsai/cudf/blob/23d24d43fac8615166c38231f13fe8751a8aec42/cpp/include/cudf/strings/convert/convert_integers.hpp#L204). 

I have created a ticket in cudf https://github.com/rapidsai/cudf/issues/14232